packages/datatrak-web-server/src/routes/UserRoute.ts

```diff
@@ -32,6 +32,12 @@ export class UserRoute extends Route<UserRequest> {
       project_id: projectId,
     } = await ctx.services.central.getUser();
 
-    return { userName: `${firstName} ${lastName}`, email, id, projectId };
+    let project = null;
+    if (projectId) {
+      const { projects } = await ctx.services.webConfig.fetchProjects();
+      project = projects.find((p: WebServerProjectRequest.ResBody) => p.id === projectId);
+    }
+
+    return { userName: `${firstName} ${lastName}`, email, id, projectId, project };
```

Comment:
I like the idea of attaching the project here since it makes life much easier on the front end